README.md

```diff
@@ -88,10 +88,12 @@ This will stop and start the polling interval in order to renew polling with new
 
 ## Events
 
-The proxy client emits two different events you can subscribe to:
+The proxy client emits those different events you can subscribe to:
```

Comment:
Nice doc update! But "those different events" sounds a little off. How about something like this?

```suggestion
The proxy client emits events that you can subscribe to. The following events are available:
```